### DISCUSSION TITLE
520.56.06

### DISCUSSION THREAD

**Discussion by aritger** [NVIDIA/open-gpu-kernel-modules#389] [Announcements]
The 520.56.06 driver and open-gpu-kernel-modules source was posted today.  It is admittedly weird that this has a lower version number than the 520.61.05 driver and source that was posted a few days ago.

These separate driver versions, and odd sequencing, are due to quirks internal to NVIDIA's release planning machinery across different organizations.

Sorry for any confusion caused by the out of order versions.

**Comment by lankylonky22** [comment]
noice, and when will the driver support vdpau or vaapi on wayland btw
